docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  app:
+    build:
+      context: .
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app/src
+    command: >
+      sh -c "python manage.py wait_for_db &&
+             python manage.py makemigrations &&
```

Comment:
why do u need this command here ?

Reply:
Because I`ve had some problems with running project, but then I realize that this is not an error and forget to remove makemigrations command(